### PR TITLE
#161382369: List Bookmarked articles

### PR DESCRIPTION
**Pull request by emukungu**
#### What does this PR do?
   List all the bookmarks of a user

#### Description of Task to be completed?
   An authenticated user should be able to see all his bookmarked articles in his/her profile and 
    navigate to them

#### How should this be manually tested?
 - Clone the repository.
 - Install application dependencies using`yarn`
 - start the server using `yarn start:dev`
 - Enter `localhost:8080/bookmarks_list` in the browser
 - A list of all bookmarked articles will be displayed
 
#### What are the relevant pivotal tracker stories?
#161382369

Screenshots
screen shot 2018-12-17 at 8 40 48 am

### CODE REVIEW

**Review by johnkegz — APPROVED** [review]
looks good

File: src/components/Header.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* eslint-disable react/no-unescaped-entities */
 import React from "react";
-import { NavLink } from "react-router-dom";
+import { NavLink, Link } from "react-router-dom";
```
Comment (shakirandagire) [inline src/components/Header.jsx]: Why did you add the Link yet you are not using it anywhere

File: src/components/ProfileImage.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React, { Component } from "react";
 import { connect } from "react-redux";
 import PropTypes from "prop-types";
-import { NavLink } from "react-router-dom";
+import { NavLink} from "react-router-dom";
```
Comment (shakirandagire) [inline src/components/ProfileImage.jsx]: Could you add the space back so as there is space between the word and the curly brace

File: tests/integerationTests/bookmarksContainer.test.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { shallow } from "enzyme";
+
+import {
+  Bookmarks,
+  CardLayout
+} from "../../src/containers/bookmarks/Bookmarks";
+
+describe("When the bookmarks component is mounted", () => {
+  describe("<Bookmarks />", () => {
```
Comment (shakirandagire) [inline tests/integerationTests/bookmarksContainer.test.jsx]: Is this second describe necessary

File: tests/integerationTests/bookmarksContainer.test.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    test("renders the component", () => {
```
Comment (shakirandagire) [inline tests/integerationTests/bookmarksContainer.test.jsx]: could you be consistent to either use test or it in the test case

Comment (emukungu) [reply tests/integerationTests/bookmarksContainer.test.jsx]: Removed the inconsistence in the tests

File: tests/integerationTests/bookmarksContainer.test.jsx (hunk 3 above)
Comment (emukungu) [reply tests/integerationTests/bookmarksContainer.test.jsx]: Removed the redundant describe function

File: src/components/Header.jsx (hunk 1 above)
Comment (emukungu) [reply src/components/Header.jsx]: Removed the redundant Link

File: src/components/ProfileImage.jsx (hunk 2 above)
Comment (emukungu) [reply src/components/ProfileImage.jsx]: Added the space to the NavLink

File: src/assets/scss/style.scss [hunk 5]
```diff
@@ -6,3 +6,6 @@ body {
   font-size: 1.6rem;
```
Comment (bozicschucky) [inline src/assets/scss/style.scss]: why do you still have two css files , may you kindly remove one you are not using .

File: src/assets/scss/style.scss [hunk 6]
```diff
@@ -1,7 +1,67 @@
 body {
   // background: #f5f5f5;
```
Comment (shakirandagire) [inline src/assets/scss/style.scss]: Could you remove commented out code

File: src/assets/scss/style.scss [hunk 7]
```diff
@@ -1,7 +1,67 @@
...
 }
+<<<<<<< HEAD
```
Comment (shakirandagire) [inline src/assets/scss/style.scss]: Could you perfect the rebase and remove this from your code ouput

File: src/assets/scss/style.scss [hunk 8]
```diff
@@ -1,7 +1,67 @@
...
+	.in-font {
+			font-size: 40px;
+
+}
+
+.image-input{
+  display: none;
+}
+.article__image{
+>>>>>>> feature(list bookmarks): List Bookmarked articles-list a users bookmarks[Finishes #161382369]feature(list bookmarks): List Bookmarked articles-list a users bookmarks[Finishes #161382369]
```
Comment (shakirandagire) [inline src/assets/scss/style.scss]: Could you also perfect the rebase and remove this from your code ouput

File: src/index.jsx [hunk 9]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import "../src/assets/css/styles.css";
+// import "../src/assets/css/styles.css";
```
Comment (shakirandagire) [inline src/index.jsx]: Could you remove commented out code

Comment (emukungu) [reply src/index.jsx]: resolved

File: src/assets/scss/style.scss (hunk 8 above)
Comment (emukungu) [reply src/assets/scss/style.scss]: resolved

File: src/assets/scss/style.scss (hunk 7 above)
Comment (emukungu) [reply src/assets/scss/style.scss]: resolved

File: src/assets/scss/style.scss (hunk 6 above)
Comment (emukungu) [reply src/assets/scss/style.scss]: resolved

File: src/assets/scss/style.scss (hunk 5 above)
Comment (emukungu) [reply src/assets/scss/style.scss]: resolved

**Review by shakirandagire — APPROVED** [review]
